packages/onboarding/src/cart/index.tsx

```diff
@@ -227,12 +227,29 @@ const addToCartAndProceed = async (
 	}
 };
 
-export async function addProductToCart(
+export async function addProductsToCart(
 	siteSlug: string,
 	flowName: string,
-	cartItem: MinimalRequestCartProduct
+	cartItems: MinimalRequestCartProduct[] | null
 ) {
-	await addToCartAndProceed( cartItem, siteSlug, flowName );
+	if ( Array.isArray( cartItems ) ) {
```

Comment:
[Note]: I added that function here because as it seems, the `addProductsToCart.actions.addProductsToCart` can take multiple products as input. 
So I don't see the value of adding them one by one, instead of bulk adding them.
I also assume that this makes a request (as it is async), so it makes more sense to add the products in bulk.

Reply:
I like the idea of adding the products in bulk.
Question: Did we try adding the products with the `addProductsToCart` returned by the hook `useShoppingCart `?
https://github.com/Automattic/wp-calypso/blob/974eef633af909d54ed034d802873c7e870cf5db/client/my-sites/checkout/composite-checkout/components/checkout-main.tsx#L191-L201

Reply:
No, I wasn't aware of that tbh. It's interesting to test this out.